.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "nixpkgs"]
 	path = nixpkgs
-	url = https://github.com/nh2/nixpkgs.git
+	url = https://github.com/NixOS/nixpkgs.git
```

Comment:
I changed the submodule here to point to `NixOS/nixpkgs` instead of `nh2/nixpkgs`.

I'm not very familiar with Git submodules, and I did this while trying to get everything working.  If necessary I can change this back.